Forge.TreeWalker.UnitTests/test/TreeWalkerUnitTests.cs

```diff
@@ -243,6 +243,20 @@ public void TestTreeWalkerSession_WalkTree_ActionHasDelay_ContinuationOnTimeout_
             Assert.AreEqual("TimeoutOnAction", actionResponse.Status, "Expected WalkTree to be successful because the TreeAction timed out but ContinuationOnTimeout flag was set.");
         }
 
+        [TestMethod]
+        public void TestTreeWalkerSession_WalkTree_ActionHasDelay_ContinuationOnRetryExhaustion_RetryPolicy_FixedCount()
+        {
+            // Initialize TreeWalkerSession with a schema containing Action with throw an excetion but RetryPolicy is fixedCount so should try.
```

Comment:
>// Initialize TreeWalkerSession with a schema containing Action with throw an excetion but RetryPolicy is fixedCount so should try. [](start = 12, length = 131)

nit: clean up the grammar on this comment a bit.

Reply:
Still same in iteration 3.

---
In reply to: [401968021](https://github.com/microsoft/Forge/pull/13#discussion_r401968021) [](ancestors = 401968021)